app/controller/team.controller.js

```diff
@@ -0,0 +1,87 @@
+const db = require('../config/db.config.js');
+const Team = db.teams;
+
+//post a Team
+exports.create = (req, res) => {
+    // Save to MYSQL
+    Team.create({
+        name: req.body.name,
+        club: req.body.club,
+        category: req.body.club,
+        profile_photo: req.body.profile_photo,
+        header_photo: req.body.header_photo
+    })
+        .then(team => {
+            //Send Created customer to Client
+            res.json(team);
+        })
+        .catch(error => res.status(400).send(error));
+};
+
+//Fetch all Teams
+exports.findAll = (req, res) => {
+    Team.findAll({
+        attributes: { exclude: ["createdAt", "updatedAt"] }
```

Comment:
I suposse that this call are only for the front-end connection for normal user interaction. If not maybe you can add this measures on histoy, logs or dashboard uses